### PR TITLE
docs to Mandarin Chinese

### PR DESCRIPTION
**Pull request by ymzcdg**
faq_general.md to Chinese
faq finished
custom_quantum_functions.md to Chinese

then I will translate docs that I need.I decide to add a new chip and operating system to QMK. I am charging myself.

Fix some bugs in English document.

thank you for working so hard!
大爱开源！

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/zh-cn/custom_quantum_functions.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+
+��ĳ����»��ͷ�ʱ�������ᱻ���á�
+
+### process_record_user()`����ʾ��ʵ��
+
+���������������¡��Զ�����һ�����`FOO`�ļ������Ϊ�����������ڰ��»س�ʱ�������
+
+```c
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {s
```
Comment (drashna) [inline docs/zh-cn/custom_quantum_functions.md]: I don't think that S is supposed to be there.
 ```suggestion
  switch (keycode) {
```

Comment (ymzcdg) [reply docs/zh-cn/custom_quantum_functions.md]: Ah, seems I must do my job quickly, I really need a new keyboard.
Thanks!

**Comment by moyi4681** [comment]
对不起，我没有你其他的联系方式，有些项目希望和你谈一下@ymzcdg. 我的邮箱moyi4681@live.cn。

**Comment by ymzcdg** [comment]
> 对不起，我没有你其他的联系方式，有些项目希望和你谈一下@ymzcdg. 我的邮箱[moyi4681@live.cn](mailto:moyi4681@live.cn)。

已通过ymzcdg@gmail.com向您发送邮件

**Comment by ymzcdg** [comment]
can i fix English documents like this?@drashna

**Comment by yanfali** [comment]
@ymzcdg of course. Good catch.

**Comment by noroadsleft** [comment]
@ymzcdg The line should read:

`Tied to one specific OS at a time (need recompilation when changing OS);`

**Comment by drashna** [comment]
Yes, please do! :)

**Comment by ymzcdg** [comment]
how can i handle 
"[Space Cadet Shift Documentation](feature_space_cadet_shift.md)" 
in
"reference_glossary.md"
the link point to 404，maybe the name of this feature was changed.

**Comment by fauxpark** [comment]
@ymzcdg the Space Cadet documentation has been consolidated into a single page: `feature_space_cadet.md`. It should just be called "Space Cadet" in that glossary page.

**Comment by ymzcdg** [comment]
> @ymzcdg the Space Cadet documentation has been consolidated into a single page: `feature_space_cadet.md`. It should just be called "Space Cadet" in that glossary page.



> @ymzcdg the Space Cadet documentation has been consolidated into a single page: `feature_space_cadet.md`. It should just be called "Space Cadet" in that glossary page.


oh! It's the name of a game

**Comment by ymzcdg** [comment]
> @ymzcdg the Space Cadet documentation has been consolidated into a single page: `feature_space_cadet.md`. It should just be called "Space Cadet" in that glossary page.

Oh! It's the name of a game, I remind my childhood. Playing 3D Pinball is interesting.

**Comment by fauxpark** [comment]
It's not the name of a game, it's a reference to this: https://en.wikipedia.org/wiki/Space-cadet_keyboard

**Comment by ymzcdg** [comment]
> It's not the name of a game, it's a reference to this: https://en.wikipedia.org/wiki/Space-cadet_keyboard

I am with very limited knowledge and scanty information. Search this name in some engine will return a game. 
Thanks!

**Pull request merged**